pkg/machines/scripts/get_libvirt_service_name.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# libvirt-bin is primary in ubuntu 1604
+NAME="`systemctl list-unit-files | grep -o libvirt-bin.service || echo libvirtd.service`"
```

Comment:
You can save the grep and the extra systemctl call below with 

    NAME=`systemctl --no-legend list-unit-files  libvirtd.service libvirt-bin.service |  head -n1 | cut -f1 -d' '`

This will prefer libvirtd.service over libvirt-bin.service even if the latter is an alias on upgrades.